client/api_challenge.go

```diff
@@ -141,7 +155,27 @@ func (c *Client) GetChallengeInfo(ctx context.Context, objectID string, pieceInd
 	return result, nil
 }
 
-// SubmitChallenge challenges the service provider data integrity, used by off-chain service greenfield-challenger.
+// SubmitChallenge challenges a storage provider's data integrity for a specific data object.
```

Comment:
SubmitChallenge - Send a transaction to challenge the SP on whether to store the committed data correctly.

We also need to add more details here, need to tell users in which case they need to use this API?

Reply:
Added more information.